apps/fipsinstall.c

```diff
@@ -424,12 +430,13 @@ int fipsinstall_main(int argc, char **argv)
                                        module_mac_len, install_mac,
                                        install_mac_len))
             goto end;
-        BIO_printf(bio_out, "INSTALL PASSED\n");
+        if (!quiet)
+            BIO_printf(bio_out, "INSTALL PASSED\n");
     }
 
     ret = 0;
 end:
-    if (ret == 1) {
+    if (ret == 1 && !quiet) {
```

Comment:
Why do you want to suppress error output?

Reply:
Most users will probably not use the -q flag. For me, the exit status suffices.

Reply:
Yeah, but what we add in our source isn't just for you. What other command do you know that suppresses *error message output* with -q? `grep` doesn't (try `grep -q foo /`)

Reply:
We don't write it just for me?  :) Shrug.  Modified commit pushed so that only the redundant "VERIFY FAILED" is suppressed with -q.  I looked, and everything that does "goto err" prints out a diagnostic first.  Unless you get the options wrong:
```
; ./apps/openssl fipsinstall -h
fipsinstall: Unknown option: -h
fipsinstall: Use -help for summary.
INSTALL FAILED
        exit 1
; 
```

Reply:
You must not of pushed the change yet?